Cargo.toml

```diff
@@ -18,9 +18,11 @@ categories = [ "data-structures", "memory-management", "rust-patterns" ]
 default = ["std"]
 std = ["alloc", "once_cell"]
 alloc = []
+generativity = ["dep:generativity"]
```

Comment:
The `dep:` syntax was only added in Rust 1.60, so this would constitute a MSRV bump for qcell.  Since it's just a pass-through anyway, is there a reason you prefer this over just the optional dependency?

Reply:
if you don't use the `dep:` syntax then you can't have a feature with the same name as a dependency. I will rename it.

Reply:
if you don't use the dep syntax, you _automatically_ get a feature with the same name as the dependency